libfreerdp/core/freerdp.c

```diff
@@ -926,7 +927,6 @@ freerdp* freerdp_new()
 	if (!instance)
 		return NULL;
 
-	winpr_InitializeSSL(WINPR_SSL_INIT_DEFAULT);
 	instance->ContextSize = sizeof(rdpContext);
```

Comment:
@akallabeth The initialization now happens in rdp_client_connect(), the problem with freerdp_new and freerdp_context_new are that they happen before the command line arguments are parsed to get the fipsmode flag. So, those wont work as is. What will be the issue without initializing it in one of those freerdp functions?